### PR TITLE
CoinbasePro  - fetchTickers

### PR DESCRIPTION
**Pull request by ttodua**
- added `fetchTickers` functionality.
- updated `parseTicker` to handle different inputs from `fetchTicker/s` ( & added `this.safeTicker`)

### CODE REVIEW

**Comment by kroitor** [comment]
Check this line: https://app.travis-ci.com/github/ccxt/ccxt/builds/243420650#L1294

**Comment by kroitor** [comment]
Actually, nevermind, it's a bug in the transpiler.

File: js/coinbasepro.js [hunk 1]
```diff
@@ -429,55 +431,95 @@ module.exports = class coinbasepro extends Exchange {
...
+        //              1639472400,
+        //              ...
+        const result = {};
+        const marketIds = Object.keys (response);
+        const delimiter = '-';
+        for (let i = 0; i < marketIds.length; i++) {
+            const marketId = marketIds[i];
+            const entry = this.safeValue (response, marketId, []);
+            const lastRow = this.safeValue (entry, 0, []);
+            const lastPrice = this.safeValue (lastRow, 4);
```
Comment (kroitor) [inline js/coinbasepro.js:517]: Fetching tickers with klines is wrong, cause OHLCV candles are aligned to timestamp boundaries, but the ticker is a floating window of 24h. I think we should not emulate it in this case.

**Comment by ttodua** [comment]
@kroitor I've update this file atm. ready to be reviewed.

**Pull request merged**